src/js/variant-widget/filters/eva-form-panel.js

```diff
@@ -193,5 +193,31 @@ EvaFormPanel.prototype = {
         });
         return form;
 
+    },
+    validatePositionFilter : function(values) {
+        var validate = true;
+        _.each(values, function (region) {
+            if(region.match(/^[\w\d]+\:\d+\-\d+/)){
```

Comment:
I have checked online and this should accept chromosomes with underscores, but what about other symbols like dashes or dots?